CHANGELOG.md

```diff
@@ -21,6 +21,28 @@ backendSrv.get(‘http://your.url/api’).then(result => {
 
 You can test your plugin with the `master` branch version of Grafana.
 
+# 6.6.1 (2020-02-06)
+
+### Features / Enhancements
+* **Quota**: Makes LDAP + Quota work for the first login of a new user. [#21949](https://github.com/grafana/grafana/pull/21949), [@xlson](https://github.com/xlson)
+
+### Bug Fixes
+* **Annotations**: Change indices and rewrites annotation find query to improve database query performance. [#21915](https://github.com/grafana/grafana/pull/21915), [@papagian](https://github.com/papagian)
```

Comment:
can you add co-authors?
[@marefr](https://github.com/marefr), [@kylebrandt](https://github.com/kylebrandt)

Reply:
Doesn't matter :smile: